### PR TITLE
aleph zero examples

### PR DESCRIPTION
**Pull request by antonefimow**

### CODE REVIEW

File: package.json
Comment (antonefimow) [reply package.json]: sure

File: src/examples/example-aleph-zero-linked.ts
Comment (MartinBechtle) [inline src/examples/example-aleph-zero-linked.ts:9]: unused import?

File: src/examples/example-aleph-zero-linked.ts
Comment (MartinBechtle) [inline src/examples/example-aleph-zero-linked.ts:14]: I see many of these are unused too

File: src/examples/example-aleph-zero-unlinked.ts
Comment (MartinBechtle) [inline src/examples/example-aleph-zero-unlinked.ts:1]: Can you add instructions on top here and in the other file like with the Terra example? And also the exact npx command to run

**Review by MartinBechtle — DISMISSED** [review]
I ran `npm install` and `npx tsx src/examples/example-aleph-zero-linked.ts` and it worked perfectly. I'll approve pending the minor changes to imports and comments, no need to wait for re-approval.

**Pull request merged**
